vnext/stubs/double-conversion/double-conversion.h

```diff
@@ -129,17 +130,27 @@ class StringToDoubleConverter {
 
   double StringToDouble(const char *s, int l, int *consumed) {
     size_t idx = 0;
-    std::string str(s, l);
-    double d = std::stod(str.c_str(), &idx);
-    *consumed = static_cast<int>(idx);
+    double d{};
+    auto ret = std::from_chars(s, s + l, d);
+    if (ret.ec == std::errc{}) {
+      *consumed = static_cast<int>(ret.ptr - s);
+    } else {
+      *consumed = 0;
+      assert(false && "Conversion to double failed");
+    }
     return d;
   }
 
-  float StringToFloat(const char *buf, int length, int *consumed) {
+  float StringToFloat(const char *s, int l, int *consumed) {
```

Comment:
It is better to keep meaningful names. One-letter names in APIs do not help readability.